explore-analyze/visualize/charts/metric-charts.md

```diff
@@ -0,0 +1,294 @@
+::::
+
+:::::
+
+## Advanced metric scenarios
+
+### Show trends in Metric charts [metric-trends]
+```{applies_to}
+stack: ga 9.1
+serverless: ga
+```
+
+When creating **Metric** visualizations with numeric data, you can add trend indicators that compare your primary metric to a secondary value. This feature displays colored badges with directional arrows to help you quickly identify whether values are increasing, decreasing, or staying the same.
+
+| Without trend | With trend |
+|--------|-------|
+| ![Secondary metric before comparison](../../images/secondary_metric_before_compare.png "title =70%") | ![Secondary metric after comparison](../../images/secondary_metric_after_compare.png "title =70%") |
+
+To add trend indicators to your metric visualization:
+
+1. Create a **Metric** visualization with a numeric primary metric.
+2. Add a secondary metric that represents the comparison value.
+
+    ::::{tip}
+    Use the `shift` parameter in formulas to compare current values against historical data. For example, if your primary metric is counting orders (based on an `order_id` field) for the current week, you can use the `count(order_id, shift='1w')` formula to compare this week's count of orders to last week's count.
+    ::::
+
+3. In the secondary metric configuration, look for the **Color by value** option. The possible choices are:
+   * **None**: No trend indicators (default)
+   * **Static**: Shows the secondary metric as a badge with a fixed color that you select
+   * **Dynamic**: Enables both color coding and directional icons based on the comparison
+
+4. Select **Dynamic** coloring. More options appear.
+
+   ![Dynamic coloring options for secondary metrics](../../images/metric-secondary-dynamic-coloring.png "=70%")
+
+5. Choose a **Color palette** that matches how you'd like to represent the comparison.
+
+6. Configure the **Display** option:
+   * **Icon**: Shows only directional arrows: ↑ for increase, ↓ for decrease, = for no change
+   * **Value**: Shows only the secondary metric value
+   * **Both**: Shows both the icon and value (default)
+
+7. The secondary metric does not automatically compare with the primary metric. Define the value to **Compare to**:
+   * **Static value**: Compares against a fixed baseline value that you specify
+   * **Primary metric**: Compares the secondary metric directly against the primary metric by displaying the result of `Primary metric - Secondary metric`. This option is only available when the primary metric is numeric. 
+   
+     When you select this option, the secondary metric is automatically updated:
+
+       * The secondary metric label changes to **Difference**. You can edit this label.
+       * If you chose a **Display** option that shows a value, the secondary metric value is automatically updated to show the difference compared to the primary metric.
+
+8. Apply your changes. 
+
+The metric visualization now shows the secondary metric as a comparison with a trend indicator.
```

Comment:
For someone with little to no data visualization skills (me), it could be helpful to describe what the end result shows, instead of what the UI displays. For example:

```suggestion
The metric visualization now shows a comparison of the primary and second metrics and trend information. 
```

Reply:
Could expand to:

The metric visualization now shows how the primary and secondary metrics compare, along with a trend indicator that highlights increases or decreases.

Reply:
- about repeating: This is by design as it's the conclusion/confirmation we've reached the end of the procedure
- about the suggestion: I think the suggestion might imply that there is only 1 metric showing in the chart as a result (the diff) instead of 1 main metric + a secondary metric displayed as a diff. I prefer to keep this as is with a focus on the secondary metric, if that makes sense.